libshortfin/docs/conf.py

```diff
@@ -0,0 +1,36 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+import os
+import sys
+
+try:
+    import _shortfin_default
+except ImportError:
+    sys.path.insert(0, os.path.abspath("../build/python/"))
+    import _shortfin_default
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "libshortfin"
+copyright = "2024, Advanced Micro Devices, Inc"
+author = "libshortfin Authors"
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+extensions = [
+    "sphinx.ext.autodoc",
+]
+
+templates_path = ["_templates"]
+exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
+
+# -- Options for HTML output -------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
+
+html_theme = "sphinx_rtd_theme"
```

Comment:
Did you try other themes? Wonder if there is something that AMD recommends

https://rocm.docs.amd.com/en/latest/contribute/toolchain.html points to https://sphinx-book-theme.readthedocs.io/en/latest/

Reply:
No not yet. I wasn't able to figure out if there is some common theme used.